src/layers/medWidgets/process/single_filter/medAbstractBiasCorrectionProcessPresenter.cpp

```diff
@@ -0,0 +1,32 @@
+/*=========================================================================
```

Comment:
Nothing done in this file. Put the empty constructor back in the header file and remove the cpp